crates/core/src/lib.rs

```diff
@@ -52,11 +52,18 @@ pub async fn start_local_surfnet(
     .await
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct PluginInfo {
```

Comment:
```suggestion
#[derive(Debug, Clone, Serialize, Deserialize)]
#[serde(rename_all = "camelCase")]
pub struct PluginInfo {
```
for most user-interfacing structs, we'll have camel case